assets/javascript/modal.js

```diff
@@ -0,0 +1,51 @@
+const modalElement = document.getElementById('modal');
+const modalOverlays = document.querySelectorAll('.modal--overlay');
+const closeBtns = document.querySelectorAll('.modal-close');
+const modalTitleElement = document.getElementById('modal--title');
+const modalIconElement = document.getElementById('modal--body--icon');
+const modalDescriptionElement = document.getElementById('modal--body--description');
+const modalBuffsElement = document.getElementById('modal--body--buffs');
+
+function openModal() {
+  if (modalElement.classList) {
```

Comment:
This is ambiguous, I think its good practice check if exist expected class
Pls give a look at this answer
https://stackoverflow.com/a/4257563/5162053